cmd/download_test.go

```diff
@@ -114,7 +114,15 @@ func TestDownload(t *testing.T) {
 		err = runDownload(cfg, flags, []string{})
 		assert.NoError(t, err)
 
-		assertDownloadedCorrectFiles(t, filepath.Join(tmpDir, tc.expectedDir), tc.requestor)
+		targetDir := filepath.Join(tmpDir, tc.expectedDir)
+		assertDownloadedCorrectFiles(t, targetDir, tc.requestor)
+
+		metadata := `{"track":"bogus-track","exercise":"bogus-exercise","id":"bogus-id","url":"","handle":"alice","is_requester":%s,"auto_approve":false}`
+		metadata = fmt.Sprintf(metadata, tc.requestor)
+		path := filepath.Join(targetDir, "bogus-track", "bogus-exercise", ".solution.json")
```

Comment:
For the future maybe in #630 there should probably be an exported variable or function for getting the path to the solution metadata file.

Reply:
Yes, agreed!

Reply:
I'm going to do that in a separate PR, as it touches other code than this download test.